tests/samplers_tests/test_samplers.py

```diff
@@ -840,17 +807,12 @@ def after_trial(
 def test_sample_single_distribution(sampler_class: Callable[[], BaseSampler]) -> None:
 
     relative_search_space = {
-        "a": UniformDistribution(low=1.0, high=1.0),
-        "b": LogUniformDistribution(low=1.0, high=1.0),
-        "c": DiscreteUniformDistribution(low=1.0, high=1.0, q=1.0),
-        "d": IntUniformDistribution(low=1, high=1),
-        "e": IntLogUniformDistribution(low=1, high=1),
-        "f": CategoricalDistribution([1]),
-        "g": FloatDistribution(low=1.0, high=1.0),
-        "h": FloatDistribution(low=1.0, high=1.0, log=True),
-        "i": FloatDistribution(low=1.0, high=1.0, step=1.0),
-        "j": IntDistribution(low=1, high=1),
-        "k": IntDistribution(low=1, high=1, log=True),
+        "a": CategoricalDistribution([1]),
+        "b": FloatDistribution(low=1.0, high=1.0),
+        "c": FloatDistribution(low=1.0, high=1.0, log=True),
+        "d": FloatDistribution(low=1.0, high=1.0, step=1.0),
+        "e": IntDistribution(low=1, high=1),
+        "f": IntDistribution(low=1, high=1, log=True),
     }
```

Comment:
I'll ask you to resolve conflicts after merging #3181. :bow:

Reply:
:smiling_face_with_tear:

Reply:
Sorry for that, thank you for the cooperation. :pray:

Reply:
No problem :D